### PR TITLE
feat: Renku 2.0 blog post

### PR DESCRIPTION
**Pull request by bethcg**
* Describes the Renku 2.0 concept 
* links to webinar 
* CTA --> user testing, feedback

### CODE REVIEW

**Review by bethcg — COMMENTED** [review]
First draft of Renku 2.0

**Comment by rokroskar** [comment]
@bethcg I added some intro text and made some edits. Btw, I don't think we should link to the google slides but instead generate a pdf, put it in the shared drive, and link to that.

File: blog/2024-03-22-renku-2-0/index.mdx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+to make sure that we stay on the right track. If you are interested in having a say
+how the project develops and contribute your thoughts to features that are in the pipeline,
+pleae drop us an email!
+
+## Further information
+
+Are you interested in learning more about Renku 2.0? If so, have a look at the following resources:
+
+- Watch our [Renku 2.0 webinar](https://www.youtube.com/channel/UCMF2tBtWU1sKWvtPl_HpI4A)
+- Check out the [roadmap](https://docs.google.com/presentation/d/1-W56dVgsR2yzrq2slA1EIyVso10jlkSazce4xOOKIZE/edit?usp=sharing)
```
Comment (lokijuhy) [inline blog/2024-03-22-renku-2-0/index.mdx:108]: I would rather link to the public roadmap in the design docs
```suggestion
- Check out our [roadmap](https://github.com/SwissDataScienceCenter/renku-design-docs/blob/main/roadmap.md)
```

File: blog/2024-03-22-renku-2-0/index.mdx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
```
Comment (lokijuhy) [inline blog/2024-03-22-renku-2-0/index.mdx:109]:
```suggestion
- Give feedback on our [design docs](https://github.com/SwissDataScienceCenter/renku-design-docs/tree/main)

```

File: blog/2024-03-22-renku-2-0/index.mdx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+- Code
+- Compute Environment
+
+Renku’s role is about making it easy for researchers to **mix and match** the elements they need to get their work done. In addition, Renku makes these elements **searchable**, encourages their **reuse** between projects, and **illuminates connections** between them (even across Projects) wherever needed. Renku accomplishes this by extracting and indexing useful metadata about each of those elements.
+
+### Data
+
+Data science projects need data! Users can connect different types of cloud data
+sources to their projects, like S3 buckets. Depending on the nature of the
+storage, they may be used in read-only or in read-write mode. However, projects
```
Comment (lokijuhy) [inline blog/2024-03-22-renku-2-0/index.mdx:70]: I think the read/write functionality is an unnecessary level of detail for this post. 
```suggestion
sources to their projects, such as S3 buckets and WebDAV resources. Projects
```

File: blog/2024-03-22-renku-2-0/index.mdx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+- If you are working on a project that uses **multiple repositories** from multiple collaborators... and you want to use all of them in 1 Renku project together.
+- If you need to showcase my project... and you want to create a **dashboard that uses a different environment configuration than the development environment**.
+
+A “Renku project” is where data science work gets done. For this, a few ingredients are required, nammely:
+
+- Data
+- Code
+- Compute Environment
+
+Renku’s role is about making it easy for researchers to **mix and match** the elements they need to get their work done. In addition, Renku makes these elements **searchable**, encourages their **reuse** between projects, and **illuminates connections** between them (even across Projects) wherever needed. Renku accomplishes this by extracting and indexing useful metadata about each of those elements.
```
Comment (lokijuhy) [inline blog/2024-03-22-renku-2-0/index.mdx:64]:
```suggestion
Renku’s role is about making it easy for researchers to **mix and match** the elements they need to get their work done. In addition, Renku makes these elements **searchable**, encourages their **reuse** between projects, and **illuminates connections** between them (even across projects) wherever needed. Renku accomplishes this by extracting and indexing useful metadata about each of those elements.
```

**Review by lokijuhy — CHANGES_REQUESTED** [review]
This looks great! I made some wordsmithing suggestions. 
More generally- I think we can use a more related cover image-? Renku 2.0 isn't about DevOps. I'll put ideas in our slack channel

File: blog/2024-03-22-renku-2-0/index.mdx [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+to make sure that we stay on the right track. If you are interested in having a say
+how the project develops and contribute your thoughts to features that are in the pipeline,
+please [drop us an email](mailto:hello@renku.io)!
+
+## Further information
+
+Are you interested in learning more about Renku 2.0? If so, have a look at the following resources:
+
+- Watch our [Renku 2.0 webinar](https://youtu.be/6J0INjPFQd4)
+- Check out the [roadmap](https://drive.google.com/file/d/1empUHHTonffW1fhdB_3DCIH-yVUz3MU8/view?usp=share_link)
```
Comment (lokijuhy) [inline blog/2024-03-22-renku-2-0/index.mdx:107]:
```suggestion
- Watch our [Renku 2.0 webinar](https://youtu.be/6J0INjPFQd4) or view the [slides](https://drive.google.com/file/d/1empUHHTonffW1fhdB_3DCIH-yVUz3MU8/view?usp=share_link)
- Check out the [Renku roadmap](https://github.com/SwissDataScienceCenter/renku-design-docs/blob/main/roadmap.md)
```

**Review by lokijuhy — CHANGES_REQUESTED** [review]
Looking great! I suggested to modify the roadmap link, and have the Webinar slides as a separate link.

**Review by lokijuhy — DISMISSED** [review]
🎆

File: blog/2024-03-22-renku-2-0/index.mdx [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+### Compute Environment
+
+Renku 2.0 will continue offering the possibility to work in the cloud, where all
+project resources are available for all collaborators. For quick, uncomplicated
+data explorations, users will be able to start immediately from pre-configured
+images, loaded with some standard libraries depending on the desired programming
+language. In addition, users will be able to define their own environments to
+use in compute sessions; these can be annotated and shared so members of a
+community can benefit from each other’s efforts and use a consistent software
+stack. Furthermore, we plan to allow users to connect their Renku projects to to
```
Comment (lokijuhy) [inline blog/2024-03-22-renku-2-0/index.mdx:92]:
```suggestion
stack. Furthermore, we plan to allow users to connect their Renku projects to
```

**Review by lokijuhy — APPROVED** [review]

**Pull request merged**